src/ucp/core/ucp_ep.c

```diff
@@ -2793,23 +2788,21 @@ int ucp_ep_do_keepalive(ucp_ep_h ep)
         worker->keepalive.lane_map &= ~UCS_BIT(lane);
     }
 
-    if (worker->keepalive.lane_map == 0) {
+    if (worker->keepalive.lane_map != 0) {
```

Comment:
add comment
/* Keepalive round is still not finished for this ep - skip ka_last_round update */